distrobox-create

```diff
@@ -118,9 +124,13 @@ if [ -z "${container_clone}" ] && [ -z "${container_image}" ]; then
 	container_image="${container_image_default}"
 fi
 
+if [ "${container_name}" == "" ]; then
+	container_name=$(basename $container_image | sed -E 's/:/_/g')
+fi
+
 # We depend on a container manager let's be sure we have it
 # First we use podman, else docker
-container_manager="podman"
+container_manager=${DB_CONTAINER_MANAGER:-"podman"}
 # Be sure we have a container manager to work with.
 if ! command -v podman >/dev/null; then
```

Comment:
This way if we set DB_CONTAINER_MANAGER=docker (for example) and we do not have podman installed, it will fail
we should at this point change it to 

` if ! command -v  "${container_manager}" >/dev/null; then`

So that the first check is always the one we declared